crutest/src/main.rs

```diff
@@ -1068,6 +1062,23 @@ async fn main() -> Result<()> {
         Workload::Version => {
             panic!("This case handled above");
         }
+        Workload::WFR => {
+            println!("Run Write-Flush-Read random IO test");
+            let count = {
+                if opt.count == 0 {
```

Comment:
Nit: it would be cleaner to store `count: Option<usize>`, then do `opt.count.unwrap_or(10)`